test/input_gen/genModelTests.py

```diff
@@ -428,3 +428,40 @@ def addition_test():
         loss_fn_str="mse"
     )
     multi_lstm_layer_return_sequence_with_batch_n(file_name="multi_lstm_return_sequence_with_batch_n.info", debug=["summary", "initial_weights", "dx", "output", "layer_name", "label","weights","gradients"],)    
+
+
+    rnn_layer_tc = lambda rnn_layer: partial(
+        record,
+        model=[
+            K.Input(batch_shape=(1, 1, 1)),
+            rnn_layer,
+            K.layers.Dense(1)
+        ],
+        optimizer=opt.SGD(learning_rate=0.1),
+        iteration=1,
+        input_shape=(1,1,1),
+        label_shape=(1,1),
+        is_onehot=False,
+        loss_fn_str="mse"
+    )
```

Comment:
Could you consider raising for more complicated scenario? shape of 1 and iteration 1 might not be sufficient.

Reply:
there will be more unittest cases followed PR.

Reply:
Awesome :)